modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/PassThroughConstants.java

```diff
@@ -234,4 +236,15 @@
 
     //Check for invalid json message by parsing the input message
     public static final String FORCE_JSON_MESSAGE_VALIDATION = "force.json.message.validation";
+
+    /**
+     * constants for the EI observability
+     */
+    public static final String CORRELATION_LOG_STATE_PROPERTY = "correlationLogState";
+    public static final String CORRELATION_ID = "correlation_id";
+    public static final String CORRELATION_LOGS_SYS_PROPERTY = "EnableCorrelationLogs";
+    public static final String CORRELATION_MDC_PROPERTY = "Correlation-ID";
+    public static final String CORRELATION_LOGGER = "CORRELATION_LOGGER";
+    public static final String CORRELATION_REQ_SEND_TO_BACKEND_TIME = "REQUEST_SENT_TIME";
```

Comment:
Lets make value of the CORRELATION_REQ_SEND_TO_BACKEND_TIME and CORRELATION_REQUEST_ARRIVED_TIME same as the variable name, basically value name is short which may override an existing one.